docs/source/common/trainer.rst

```diff
@@ -241,6 +241,25 @@ Example::
 
     Trainer(accelerator=MyOwnAcc())
 
+.. note::
+
+    If the ``devices`` flag is not defined, it will assume ``devices`` to be ``"auto"`` and fetch the ``auto_device_count``
+    from the accelerator.
+
+.. code-block:: python
```

Comment:
I think if you indent it, then it will appear under the note section.
like this: https://torchmetrics.readthedocs.io/en/latest/pages/lightning.html

Reply:
It was intentional, don't want under `notes` section

Reply:
why is that? isn't the example reflecting what's mentioned in the note?